src/util/prefer-v2-deployment.ts

```diff
@@ -2,7 +2,7 @@ import { join } from 'path';
 import { exists } from 'fs-extra';
 import { Config } from '../types';
 import { Package } from './dev/types';
-import {CantParseJSONFile } from './errors-ts';
+import { CantParseJSONFile } from './errors-ts';
```

Comment:
Can you run the `prettier` command on this file? (I'm not sure why `format-modified` pre-commit hook doesn't work 🤔)